docs/integration/php.md

```diff
@@ -88,56 +81,53 @@ LoadModule fastcgi_module modules/mod_fastcgi.so
 </IfModule>
 ```
 
-Run the setup command using HTTP URLs for status pages:
-
-```sh
-  # sematext-httpd-setup YOUR_MONITORING_TOKEN_HERE HTTPD_STATUS_URL PHP_FPM_STATUS_URL
-  sudo sematext-httpd-setup YOUR_MONITORING_TOKEN_HERE -u http://localhost/server-status http://localhost/status
-```
+In this case, parameter `ST_MONITOR_PHPFPM_STATUS_URL` should have a value `http://localhost/status`.
 
 ## Integration with Nginx
 
-** Install sematext-agent-nginx via npm **
-```sh
-sudo npm i sematext-agent-httpd -g
-```
+Sematext Agent can be installed to monitor [Apache](./apache).
+
+When using [Autodiscovery](../monitoring/autodiscovery), Sematext Agent will automatically discover PHP-FPM status
+URL and will gather additional metrics from it.
+
+Otherwise, when using manual installation instruction, additional parameter `ST_MONITOR_PHPFPM_STATUS_URL` can be passed
+to setup scripts to define how to access PHP-FPM metrics.
 
 ** Setup Nginx Agent with php-fpm UNIX socket (recommended) **
 
-Run the service setup for the PHP-FPM monitoring agent. Pass the
-Sematext Monitoring App token (aka SPM token), Nginx status URL, and
-the PHP-FPM status URL to the setup command:
-```sh
-sematext-nginx-setup -t YOUR_MONITORING_TOKEN_HERE -n http://localhost/nginx_status -p http://unix:/var/run/php-fpm.sock:/status
-```
+Add a parameter `ST_MONITOR_PHPFPM_STATUS_URL` with a value
+`http://unix:/var/run/php-fpm.sock:/status`. E.g., when using Ubuntu instructions add
+`--ST_MONITOR_PHPFPM_STATUS_URL=http://unix:/var/run/php-fpm.sock:/status` param when calling
+`sudo bash /opt/spm/bin/setup-sematext ...` command.
 
 ** Setup with PHP-FPM status page via HTTP **
 
 In some scenarios, e.g. in Docker containers, the monitoring agent
 might not have access to the local UNIX socket. In such cases the
-PHP-FPM status page needs to be exposed via Nginx.  To expose the
-PHP-FPM status page via Nginx change the Nginx configuration
-```/etc/nginx/sites-enabled/default```:
+PHP-FPM status page needs to be exposed via Apache httpd.  To expose
```

Comment:
Should it be Apache in this section about Nginx?

Reply:
Yeah, bad copy-paste it seems.